src/frontend/state/GlobalState.tsx

```diff
@@ -56,14 +58,12 @@ interface Props {
   children: React.ReactNode
   i18n: i18n
   t: T
+  epic: EpicState
+  setEpic: SetterOrUpdater<EpicState>
```

Comment:
we need to add these props temporarily until we migrate everything that needs getters and setters inside GlobalState